acceptanceTests/tap_test.go

```diff
@@ -973,3 +973,197 @@ func TestTapDumpLogs(t *testing.T) {
 		return
 	}
 }
+
+func TestTapDaemon(t *testing.T) {
```

Comment:
Give it a more detailed name.
This function doesn't test all of daemon mode capabilities and edge cases.
It only tests that traffic is recorded. For example, this test doesn't compare the recorded traffic to the transmitted messages, and it doesn't check that pods are filtered correctly.
This test is good in itself. I think that the names should be accurate so that we are clear on what the tests cover.
Maybe `TestDaemonSeeTraffic` or `TestDaemonListEntries`?

Reply:
Was just following naming convention, this is a daemon version of the test `TestTap`, didnt realize this would be misleading

Reply:
I think it is 🤷‍♂️ 
You don't have to fix all of the tests, just the new ones.